packages/airnode-examples/README.md

```diff
@@ -93,9 +93,9 @@ deploy the contracts and make transactions, so make sure it is funded. The recom
 
 ### 4. Deploy the RRP contract
 
-The [RRP contract](https://docs.api3.org/airnode/v0.2/concepts/#airnoderrp-sol) is a contract through which the
-[requester](https://docs.api3.org/airnode/v0.2/concepts/requester.html) triggers a request for Airnode. This contract is
-common for all Airnodes and requesters on a chain.
+The [RRP contract](https://docs.api3.org/airnode/v0.3/concepts/#airnoderrp-sol) is a contract through which the
```

Comment:
Can't we migrate all of them to latest? I know Warren wanted to look at how it should could be done on the docs side. 

This feels tiring to update after every release...

Reply:
I agree it's tiring, but presently links with anchors don't work with the /latest redirect as mentioned in the PR description. Warren is taking a look, but in the meantime let's merge as this will get rid of outdated links, and I can update links with anchors (and others referring to v0.3) after it's addressed.